### ISSUE TITLE
Operator processes only 6 Scylla clusters at a time making other wait until the full completion of the previous set

### ISSUE THREAD

**Issue by vponomaryov** [scylladb/scylla-operator#997]
**Describe the bug**
When we schedule creation of lots of Scylla clusters, bigger than 6, then we see that operators picks up for processing
first 6 Scylla clusters, waits for it's completion and only then picks up other 6.
So, if we schedule 14 clusters then we have 3 stages of Scylla clusters creations - 6 ,6 and 2 clusters at a time.
Pay attention at the pod creation time in the following list of pods (`14 clusters`):
```
$ kubectl get pods -A -o wide
NAMESPACE                     NAME                                                             READY   STATUS        RESTARTS   AGE    IP           NODE
cert-manager                  cert-manager-759575d694-xjrv4                                    1/1     Running       0          145m   10.0.0.10    ip-10-0-1-226.ec2.internal
cert-manager                  cert-manager-cainjector-77977c9f5c-8xghj                         1/1     Running       0          145m   10.0.0.62    ip-10-0-1-136.ec2.internal
cert-manager                  cert-manager-webhook-746769d4-vgrgz                              1/1     Running       0          145m   10.0.2.16    ip-10-0-1-226.ec2.internal
default                       cpu-policy-c8vn6                                                 1/1     Running       1          144m   10.0.3.217   ip-10-0-0-158.ec2.internal
default                       cpu-policy-jwnsm                                                 1/1     Running       1          144m   10.0.3.69    ip-10-0-3-118.ec2.internal
default                       cpu-policy-ns85b                                                 1/1     Running       1          144m   10.0.3.15    ip-10-0-3-158.ec2.internal
default                       local-volume-provisioner-5szq5                                   1/1     Running       0          144m   10.0.2.65    ip-10-0-3-118.ec2.internal
default                       local-volume-provisioner-bctcb                                   1/1     Running       0          144m   10.0.2.15    ip-10-0-0-158.ec2.internal
default                       local-volume-provisioner-zdg9b                                   1/1     Running       0          144m   10.0.2.198   ip-10-0-3-158.ec2.internal
default                       node-setup-5mzsh                                                 2/2     Running       0          144m   10.0.0.216   ip-10-0-0-158.ec2.internal
default                       node-setup-tgj8g                                                 2/2     Running       0          144m   10.0.1.153   ip-10-0-3-118.ec2.internal
default                       node-setup-zrgtb                                                 2/2     Running       0          144m   10.0.3.92    ip-10-0-3-158.ec2.internal
kube-system                   aws-node-2wm24                                                   1/1     Running       0          145m   10.0.3.118   ip-10-0-3-118.ec2.internal
... 17 more such pods ...
kube-system                   aws-node-zxpz8                                                   1/1     Running       0          145m   10.0.1.136   ip-10-0-1-136.ec2.internal
kube-system                   coredns-75b7767488-b65rm                                         1/1     Running       0          147m   10.0.3.133   ip-10-0-1-226.ec2.internal
kube-system                   coredns-75b7767488-dnqs2                                         1/1     Running       0          147m   10.0.3.123   ip-10-0-1-136.ec2.internal
kube-system                   kube-proxy-52hrk                                                 1/1     Running       0          146m   10.0.2.111   ip-10-0-2-111.ec2.internal
... 17 more such pods ...
kube-system                   kube-proxy-zpfwf                                                 1/1     Running       0          146m   10.0.1.178   ip-10-0-1-178.ec2.internal
sct-loaders-10                sct-loaders-10-0                                                 1/1     Running       0          112m   10.0.3.53    ip-10-0-0-191.ec2.internal
sct-loaders-11                sct-loaders-11-0                                                 1/1     Running       0          112m   10.0.0.212   ip-10-0-2-111.ec2.internal
sct-loaders-12                sct-loaders-12-0                                                 1/1     Running       0          112m   10.0.2.12    ip-10-0-3-229.ec2.internal
sct-loaders-13                sct-loaders-13-0                                                 1/1     Running       0          112m   10.0.2.98    ip-10-0-1-227.ec2.internal
sct-loaders-14                sct-loaders-14-0                                                 1/1     Running       0          112m   10.0.1.252   ip-10-0-1-178.ec2.internal
sct-loaders-2                 sct-loaders-2-0                                                  1/1     Running       0          113m   10.0.2.172   ip-10-0-1-42.ec2.internal
sct-loaders-3                 sct-loaders-3-0                                                  1/1     Running       0          113m   10.0.0.194   ip-10-0-3-14.ec2.internal
sct-loaders-4                 sct-loaders-4-0                                                  1/1     Running       0          113m   10.0.3.38    ip-10-0-2-206.ec2.internal
sct-loaders-5                 sct-loaders-5-0                                                  1/1     Running       0          113m   10.0.0.253   ip-10-0-2-88.ec2.internal
sct-loaders-6                 sct-loaders-6-0                                                  1/1     Running       0          113m   10.0.1.134   ip-10-0-1-122.ec2.internal
sct-loaders-7                 sct-loaders-7-0                                                  1/1     Running       0          113m   10.0.1.144   ip-10-0-1-211.ec2.internal
sct-loaders-8                 sct-loaders-8-0                                                  1/1     Running       0          113m   10.0.3.59    ip-10-0-0-14.ec2.internal
sct-loaders-9                 sct-loaders-9-0                                                  1/1     Running       0          112m   10.0.1.164   ip-10-0-3-213.ec2.internal
sct-loaders                   sct-loaders-0                                                    1/1     Running       0          113m   10.0.0.143   ip-10-0-3-50.ec2.internal
scylla-10                     sct-cluster-10-us-east1-b-us-east1-0                             2/2     Running       1          127m   10.0.2.77    ip-10-0-3-118.ec2.internal
scylla-10                     sct-cluster-10-us-east1-b-us-east1-1                             2/2     Running       1          124m   10.0.3.77    ip-10-0-3-158.ec2.internal
scylla-10                     sct-cluster-10-us-east1-b-us-east1-2                             2/2     Running       1          120m   10.0.0.186   ip-10-0-3-158.ec2.internal
scylla-11                     sct-cluster-11-us-east1-b-us-east1-0                             2/2     Running       1          127m   10.0.0.225   ip-10-0-0-158.ec2.internal
scylla-11                     sct-cluster-11-us-east1-b-us-east1-1                             2/2     Running       1          124m   10.0.0.109   ip-10-0-0-158.ec2.internal
scylla-11                     sct-cluster-11-us-east1-b-us-east1-2                             2/2     Running       1          120m   10.0.0.146   ip-10-0-0-158.ec2.internal
scylla-12                     sct-cluster-12-us-east1-b-us-east1-0                             2/2     Running       1          127m   10.0.0.183   ip-10-0-3-158.ec2.internal
scylla-12                     sct-cluster-12-us-east1-b-us-east1-1                             2/2     Running       1          124m   10.0.2.178   ip-10-0-3-118.ec2.internal
scylla-12                     sct-cluster-12-us-east1-b-us-east1-2                             2/2     Running       1          120m   10.0.1.62    ip-10-0-3-118.ec2.internal
scylla-13                     sct-cluster-13-us-east1-b-us-east1-0                             2/2     Running       1          113m   10.0.2.186   ip-10-0-3-158.ec2.internal
scylla-13                     sct-cluster-13-us-east1-b-us-east1-1                             2/2     Running       1          110m   10.0.2.125   ip-10-0-0-158.ec2.internal
scylla-13                     sct-cluster-13-us-east1-b-us-east1-2                             2/2     Running       1          106m   10.0.0.108   ip-10-0-3-118.ec2.internal
scylla-14                     sct-cluster-14-us-east1-b-us-east1-0                             2/2     Running       1          113m   10.0.0.237   ip-10-0-3-118.ec2.internal
scylla-14                     sct-cluster-14-us-east1-b-us-east1-1                             2/2     Running       1          110m   10.0.1.151   ip-10-0-0-158.ec2.internal
scylla-14                     sct-cluster-14-us-east1-b-us-east1-2                             2/2     Running       1          106m   10.0.1.90    ip-10-0-3-158.ec2.internal
scylla-2                      sct-cluster-2-us-east1-b-us-east1-0                              2/2     Running       1          141m   10.0.2.236   ip-10-0-0-158.ec2.internal
scylla-2                      sct-cluster-2-us-east1-b-us-east1-1                              2/2     Running       1          138m   10.0.0.179   ip-10-0-3-158.ec2.internal
scylla-2                      sct-cluster-2-us-east1-b-us-east1-2                              2/2     Running       1          134m   10.0.2.84    ip-10-0-3-118.ec2.internal
scylla-3                      sct-cluster-3-us-east1-b-us-east1-0                              2/2     Running       1          141m   10.0.1.85    ip-10-0-3-118.ec2.internal
scylla-3                      sct-cluster-3-us-east1-b-us-east1-1                              2/2     Running       1          138m   10.0.2.30    ip-10-0-3-118.ec2.internal
scylla-3                      sct-cluster-3-us-east1-b-us-east1-2                              2/2     Running       1          134m   10.0.3.235   ip-10-0-3-118.ec2.internal
scylla-4                      sct-cluster-4-us-east1-b-us-east1-0                              2/2     Running       1          141m   10.0.2.159   ip-10-0-0-158.ec2.internal
scylla-4                      sct-cluster-4-us-east1-b-us-east1-1                              2/2     Running       1          138m   10.0.0.40    ip-10-0-0-158.ec2.internal
scylla-4                      sct-cluster-4-us-east1-b-us-east1-2                              2/2     Running       1          134m   10.0.3.209   ip-10-0-3-158.ec2.internal
scylla-5                      sct-cluster-5-us-east1-b-us-east1-0                              2/2     Running       1          141m   10.0.1.145   ip-10-0-3-118.ec2.internal
scylla-5                      sct-cluster-5-us-east1-b-us-east1-1                              2/2     Running       1          139m   10.0.3.183   ip-10-0-3-158.ec2.internal
scylla-5                      sct-cluster-5-us-east1-b-us-east1-2                              2/2     Running       1          134m   10.0.3.47    ip-10-0-0-158.ec2.internal
scylla-6                      sct-cluster-6-us-east1-b-us-east1-0                              2/2     Running       1          141m   10.0.1.203   ip-10-0-3-158.ec2.internal
scylla-6                      sct-cluster-6-us-east1-b-us-east1-1                              2/2     Running       1          138m   10.0.0.168   ip-10-0-3-118.ec2.internal
scylla-6                      sct-cluster-6-us-east1-b-us-east1-2                              2/2     Running       1          134m   10.0.1.208   ip-10-0-3-158.ec2.internal
scylla-7                      sct-cluster-7-us-east1-b-us-east1-0                              2/2     Running       1          128m   10.0.0.229   ip-10-0-0-158.ec2.internal
scylla-7                      sct-cluster-7-us-east1-b-us-east1-1                              2/2     Running       1          125m   10.0.2.103   ip-10-0-3-158.ec2.internal
scylla-7                      sct-cluster-7-us-east1-b-us-east1-2                              2/2     Running       1          120m   10.0.3.89    ip-10-0-3-118.ec2.internal
scylla-8                      sct-cluster-8-us-east1-b-us-east1-0                              2/2     Running       1          127m   10.0.0.178   ip-10-0-3-158.ec2.internal
scylla-8                      sct-cluster-8-us-east1-b-us-east1-1                              2/2     Running       1          124m   10.0.0.47    ip-10-0-0-158.ec2.internal
scylla-8                      sct-cluster-8-us-east1-b-us-east1-2                              2/2     Running       1          120m   10.0.0.213   ip-10-0-3-158.ec2.internal
scylla-9                      sct-cluster-9-us-east1-b-us-east1-0                              2/2     Running       1          127m   10.0.1.12    ip-10-0-3-118.ec2.internal
scylla-9                      sct-cluster-9-us-east1-b-us-east1-1                              2/2     Running       1          124m   10.0.0.87    ip-10-0-3-118.ec2.internal
scylla-9                      sct-cluster-9-us-east1-b-us-east1-2                              2/2     Running       1          120m   10.0.2.200   ip-10-0-0-158.ec2.internal
scylla-operator-node-tuning   perftune-containers-1df0e5a4-0704-4b96-a13e-afab1f3cebe1-4f77b   0/1     Terminating   0          2s     <none>       ip-10-0-3-118.ec2.internal
scylla-operator-node-tuning   perftune-containers-1df0e5a4-0704-4b96-a13e-afab1f3cebe1-9s5gz   0/1     Terminating   0          0s     <none>       ip-10-0-3-118.ec2.internal
scylla-operator-node-tuning   perftune-containers-1df0e5a4-0704-4b96-a13e-afab1f3cebe1-ppsn2   0/1     Terminating   0          6s     10.0.3.118   ip-10-0-3-118.ec2.internal
scylla-operator-node-tuning   perftune-containers-6c8f1835-70b7-4c58-a1bb-bb4c55b02b5b-844wk   0/1     Terminating   0          0s     <none>       ip-10-0-3-158.ec2.internal
scylla-operator-node-tuning   perftune-containers-6c8f1835-70b7-4c58-a1bb-bb4c55b02b5b-hxgr9   0/1     Terminating   0          3s     10.0.3.158   ip-10-0-3-158.ec2.internal
scylla-operator-node-tuning   perftune-containers-6c8f1835-70b7-4c58-a1bb-bb4c55b02b5b-lzxpn   0/1     Terminating   0          6s     10.0.3.158   ip-10-0-3-158.ec2.internal
scylla-operator-node-tuning   perftune-containers-6c8f1835-70b7-4c58-a1bb-bb4c55b02b5b-rhs4z   0/1     Terminating   0          5s     10.0.3.158   ip-10-0-3-158.ec2.internal
scylla-operator-node-tuning   perftune-containers-6c8f1835-70b7-4c58-a1bb-bb4c55b02b5b-xfxjl   0/1     Terminating   0          2s     <none>       ip-10-0-3-158.ec2.internal
scylla-operator-node-tuning   perftune-containers-6c8f1835-70b7-4c58-a1bb-bb4c55b02b5b-zjz4b   0/1     Terminating   0          2s     10.0.3.158   ip-10-0-3-158.ec2.internal
scylla-operator-node-tuning   perftune-containers-fb4cfe21-d758-4f47-a7e8-5f31c43108ae-2zq47   0/1     Terminating   0          35s    10.0.0.158   ip-10-0-0-158.ec2.internal
scylla-operator-node-tuning   perftune-containers-fb4cfe21-d758-4f47-a7e8-5f31c43108ae-749sd   0/1     Terminating   0          0s     <none>       ip-10-0-0-158.ec2.internal
scylla-operator-node-tuning   perftune-containers-fb4cfe21-d758-4f47-a7e8-5f31c43108ae-94v7n   0/1     Terminating   0          7s     10.0.0.158   ip-10-0-0-158.ec2.internal
scylla-operator-node-tuning   perftune-containers-fb4cfe21-d758-4f47-a7e8-5f31c43108ae-pdjj2   0/1     Terminating   0          5s     10.0.0.158   ip-10-0-0-158.ec2.internal
scylla-operator-node-tuning   perftune-containers-fb4cfe21-d758-4f47-a7e8-5f31c43108ae-pmhcl   0/1     Terminating   0          2s     <none>       ip-10-0-0-158.ec2.internal
scylla-operator-node-tuning   perftune-containers-fb4cfe21-d758-4f47-a7e8-5f31c43108ae-t6bc4   0/1     Terminating   0          3s     10.0.0.158   ip-10-0-0-158.ec2.internal
scylla-operator-node-tuning   perftune-node-1df0e5a4-0704-4b96-a13e-afab1f3cebe1-g86m7         0/1     Completed     0          141m   10.0.3.118   ip-10-0-3-118.ec2.internal
scylla-operator-node-tuning   perftune-node-6c8f1835-70b7-4c58-a1bb-bb4c55b02b5b-xzmv2         0/1     Completed     0          141m   10.0.3.158   ip-10-0-3-158.ec2.internal
scylla-operator-node-tuning   perftune-node-fb4cfe21-d758-4f47-a7e8-5f31c43108ae-qxmg4         0/1     Completed     0          141m   10.0.0.158   ip-10-0-0-158.ec2.internal
scylla-operator-node-tuning   scylla-layout-nodes-tuning-bdwvk                                 1/1     Running       0          143m   10.0.0.158   ip-10-0-0-158.ec2.internal
scylla-operator-node-tuning   scylla-layout-nodes-tuning-jxxld                                 1/1     Running       0          143m   10.0.3.118   ip-10-0-3-118.ec2.internal
scylla-operator-node-tuning   scylla-layout-nodes-tuning-nr6w2                                 1/1     Running       0          143m   10.0.3.158   ip-10-0-3-158.ec2.internal
scylla-operator               scylla-operator-5d98c9bf6c-7zcbc                                 1/1     Running       0          144m   10.0.0.188   ip-10-0-1-136.ec2.internal
scylla-operator               scylla-operator-5d98c9bf6c-mmtq7                                 1/1     Running       0          144m   10.0.2.101   ip-10-0-1-226.ec2.internal
scylla-operator               webhook-server-675677d8f4-nqz4q                                  1/1     Running       0          144m   10.0.1.204   ip-10-0-1-226.ec2.internal
scylla-operator               webhook-server-675677d8f4-zj6lv                                  1/1     Running       0          144m   10.0.2.190   ip-10-0-1-136.ec2.internal
scylla                        sct-cluster-us-east1-b-us-east1-0                                2/2     Running       1          141m   10.0.2.90    ip-10-0-3-158.ec2.internal
scylla                        sct-cluster-us-east1-b-us-east1-1                                2/2     Running       1          138m   10.0.0.80    ip-10-0-0-158.ec2.internal
scylla                        sct-cluster-us-east1-b-us-east1-2                                2/2     Running       1          134m   10.0.0.230   ip-10-0-0-158.ec2.internal
```

**To Reproduce**
1. Deploy operator
2. Schedule creation of 7+ Scylla clusters at a time
3. See the described problem

**Expected behavior**
There should not be such a restriction. Scylla pods must be scheduled for creation not waiting for completion of previous 6 Scylla clusters's pods.

**Logs**
Please, provide `kubectl get events`, `kubectl logs -n scylla pod-name` and other relevant information.

**Environment:**
 - Platform: EKS (platform is not expected to make any effect)
 - Kubernetes version: v1.20.15-eks-a64ea69
 - Scylla version: 4.6.3 (it is not expected to make any effect)
 - Scylla-operator version: e.g.: v1.7.2 (1.8-dev is expected to be affected too)

**Comment by zimnx** [comment]
Number of concurrent workers is configurable in scylla-operator. 
The default value is 5 which is close to what you're seeing. 
Try to increase it by setting `--concurrent-syncs` parameter and see if it helps.

**Comment by vponomaryov** [comment]
> Number of concurrent workers is configurable in scylla-operator. The default value is 5 which is close to what you're seeing. Try to increase it by setting `--concurrent-syncs` parameter and see if it helps.
> 
> Fix for #994 may also help, if issue lays in queue being clogged.

`scylla-operator` helm chart doesn't support providing this option. Sure, I can do it after operator deployment, but users won't do it.

**Comment by vponomaryov** [comment]
> > Number of concurrent workers is configurable in scylla-operator. The default value is 5 which is close to what you're seeing. Try to increase it by setting `--concurrent-syncs` parameter and see if it helps.
> > Fix for #994 may also help, if issue lays in queue being clogged.
> 
> `scylla-operator` helm chart doesn't support providing this option. Sure, I can do it after operator deployment, but users won't do it. Also, why is it set to such a small value by default? Will operator have any problems syncing 20 clusters? 30?

Tried setting following:

```
...
    Args:
      operator
      --loglevel=4
      --concurrent-syncs=14
...
```
No effect. At all.

**Comment by zimnx** [comment]
5 should be enough, we shouldn't have that much work. I think queue got clogged, and #994 will fix it aswell. I tried spawning 10 clusters with this fix, and all clusters were reconciled in parallel:
```
basic--4-t2fqz-us-east-1-us-east-1a-0   0/2     Pending   0          0s
basic--1-rws59-us-east-1-us-east-1a-0   0/2     Pending   0          0s
basic--3-lz7sf-us-east-1-us-east-1a-0   0/2     Pending   0          0s
basic--2-ft8tr-us-east-1-us-east-1a-0   0/2     Pending   0          0s
basic--0-j57hn-us-east-1-us-east-1a-0   0/2     Pending   0          0s
basic--4-t2fqz-us-east-1-us-east-1a-0   0/2     Pending   0          0s
basic--1-rws59-us-east-1-us-east-1a-0   0/2     Pending   0          0s
basic--3-lz7sf-us-east-1-us-east-1a-0   0/2     Pending   0          0s
basic--2-ft8tr-us-east-1-us-east-1a-0   0/2     Pending   0          0s
basic--0-j57hn-us-east-1-us-east-1a-0   0/2     Pending   0          0s
basic--4-t2fqz-us-east-1-us-east-1a-0   0/2     Pending   0          1s
basic--2-ft8tr-us-east-1-us-east-1a-0   0/2     Pending   0          1s
basic--3-lz7sf-us-east-1-us-east-1a-0   0/2     Pending   0          1s
basic--0-j57hn-us-east-1-us-east-1a-0   0/2     Pending   0          1s
basic--1-rws59-us-east-1-us-east-1a-0   0/2     Pending   0          1s
basic--4-t2fqz-us-east-1-us-east-1a-0   0/2     Init:0/2   0          1s
basic--2-ft8tr-us-east-1-us-east-1a-0   0/2     Init:0/2   0          2s
basic--3-lz7sf-us-east-1-us-east-1a-0   0/2     Init:0/2   0          3s
basic--0-j57hn-us-east-1-us-east-1a-0   0/2     Init:0/2   0          4s
basic--1-rws59-us-east-1-us-east-1a-0   0/2     Init:0/2   0          5s
basic--4-t2fqz-us-east-1-us-east-1a-0   0/2     Init:0/2   0          6s
basic--2-ft8tr-us-east-1-us-east-1a-0   0/2     Init:0/2   0          8s
basic--3-lz7sf-us-east-1-us-east-1a-0   0/2     Init:0/2   0          8s
basic--0-j57hn-us-east-1-us-east-1a-0   0/2     Init:0/2   0          10s
basic--1-rws59-us-east-1-us-east-1a-0   0/2     Init:0/2   0          11s
basic--5-vrf9v-us-east-1-us-east-1a-0   0/2     Pending    0          0s
basic--5-vrf9v-us-east-1-us-east-1a-0   0/2     Pending    0          0s
basic--4-t2fqz-us-east-1-us-east-1a-0   0/2     Init:1/2   0          12s
basic--6-x22f4-us-east-1-us-east-1a-0   0/2     Pending    0          0s
basic--2-ft8tr-us-east-1-us-east-1a-0   0/2     Init:1/2   0          12s
basic--6-x22f4-us-east-1-us-east-1a-0   0/2     Pending    0          0s
basic--7-xgcnj-us-east-1-us-east-1a-0   0/2     Pending    0          0s
basic--7-xgcnj-us-east-1-us-east-1a-0   0/2     Pending    0          0s
basic--9-5pf2j-us-east-1-us-east-1a-0   0/2     Pending    0          0s
basic--8-qzmfg-us-east-1-us-east-1a-0   0/2     Pending    0          0s
basic--9-5pf2j-us-east-1-us-east-1a-0   0/2     Pending    0          0s
basic--8-qzmfg-us-east-1-us-east-1a-0   0/2     Pending    0          0s
basic--3-lz7sf-us-east-1-us-east-1a-0   0/2     Init:1/2   0          13s
basic--5-vrf9v-us-east-1-us-east-1a-0   0/2     Pending    0          2s
basic--7-xgcnj-us-east-1-us-east-1a-0   0/2     Pending    0          2s
basic--9-5pf2j-us-east-1-us-east-1a-0   0/2     Pending    0          2s
basic--8-qzmfg-us-east-1-us-east-1a-0   0/2     Pending    0          2s
basic--6-x22f4-us-east-1-us-east-1a-0   0/2     Pending    0          2s
basic--0-j57hn-us-east-1-us-east-1a-0   0/2     Init:1/2   0          14s
basic--4-t2fqz-us-east-1-us-east-1a-0   0/2     PodInitializing   0          16s
basic--3-lz7sf-us-east-1-us-east-1a-0   0/2     PodInitializing   0          17s
basic--2-ft8tr-us-east-1-us-east-1a-0   0/2     PodInitializing   0          18s
basic--9-5pf2j-us-east-1-us-east-1a-0   0/2     Init:0/2          0          6s
basic--0-j57hn-us-east-1-us-east-1a-0   0/2     PodInitializing   0          19s
basic--1-rws59-us-east-1-us-east-1a-0   0/2     PodInitializing   0          20s
basic--8-qzmfg-us-east-1-us-east-1a-0   0/2     Init:0/2          0          9s
basic--5-vrf9v-us-east-1-us-east-1a-0   0/2     Init:0/2          0          11s
basic--7-xgcnj-us-east-1-us-east-1a-0   0/2     Init:0/2          0          12s
basic--6-x22f4-us-east-1-us-east-1a-0   0/2     Init:0/2          0          12s
basic--7-xgcnj-us-east-1-us-east-1a-0   0/2     Init:0/2          0          13s
basic--7-xgcnj-us-east-1-us-east-1a-0   0/2     Init:1/2          0          13s
basic--5-vrf9v-us-east-1-us-east-1a-0   0/2     Init:0/2          0          14s
basic--9-5pf2j-us-east-1-us-east-1a-0   0/2     Init:0/2          0          15s
basic--8-qzmfg-us-east-1-us-east-1a-0   0/2     Init:0/2          0          16s
basic--5-vrf9v-us-east-1-us-east-1a-0   0/2     Init:1/2          0          17s
basic--9-5pf2j-us-east-1-us-east-1a-0   0/2     Init:1/2          0          17s
basic--5-vrf9v-us-east-1-us-east-1a-0   0/2     PodInitializing   0          18s
basic--9-5pf2j-us-east-1-us-east-1a-0   0/2     PodInitializing   0          18s

**Comment by vponomaryov** [comment]
The problem was with automation. Closing it.
